CHANGELOG.rst

```diff
@@ -6,6 +6,8 @@ Changelog
 
 .. note:: This version is not yet released and is under active development.
 
+* Fix a regression in parsing requirements with multiple versions and
+  whitespace, like ``foo>=2.0, !=2.1``.
```

Comment:
You might want to add:

    (:issue:`65`)

To link to the corresponding issue.